Source/Demo/DemoQuest/Rooms/EmptyStreet.cs

```diff
@@ -76,7 +76,10 @@ private void subscribeEvents()
 			_room.Edges.Right.OnEdgeCrossed.Subscribe(onRightEdgeCrossed);
 			_room.Events.OnBeforeFadeIn.Subscribe(onBeforeFadeIn);
 			_room.Events.OnAfterFadeIn.Subscribe(onAfterFadeIn);
-            _bottle?.Interactions.OnInteract(AGSInteractions.INTERACT).SubscribeToAsync(onBottleInteract);
+            if (_bottle != null)
```

Comment:
The Elvis operator should still work here, no?
`_bottle?.GetComponent<IHotspotComponent>().Interactions.OnInteract(AGSInteractions.INTERACT).SubscribeToAsync(onBottleInteract);